demo/site/src/blocks/DamVideoBlock.tsx

```diff
@@ -1,16 +1,34 @@
 import { PreviewSkeleton, PropsWithData, withPreview } from "@comet/cms-site";
 import { DamVideoBlockData } from "@src/blocks.generated";
 import * as React from "react";
+import styled from "styled-components";
+
+export const VideoContainer = styled.div`
```

Comment:
```suggestion
const VideoContainer = styled.div`
```

No need to export this.

Reply:
https://github.com/vivid-planet/comet/pull/2068/commits/8887fe044daddee7e4711c2ee20f34a86908166d